### DISCUSSION TITLE
Github page only half works?

### DISCUSSION THREAD

**Discussion by moenegm** [community/community#23985] [Other Feature Feedback, Questions, & Ideas]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/5/5/55ae289ce15b7a9781a96f40930196d809987dfb.jpeg" data-download-href="/uploads/short-url/cdXIwC1Na1gDqdMH2czk0aWFIhl.jpeg?dl=1" title="Screen Shot 2021-03-15 at 12.01.38 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/44214327/181080507-2aad4192-2bbe-4215-9ae9-ed41be9f006e.jpeg" alt="Screen Shot 2021-03-15 at 12.01.38 PM" data-base62-sha1="cdXIwC1Na1gDqdMH2czk0aWFIhl" width="690" height="338" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/5/55ae289ce15b7a9781a96f40930196d809987dfb_2_690x338.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/5/5/55ae289ce15b7a9781a96f40930196d809987dfb_2_1035x507.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/5/5/55ae289ce15b7a9781a96f40930196d809987dfb_2_1380x676.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/5/55ae289ce15b7a9781a96f40930196d809987dfb_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2021-03-15 at 12.01.38 PM</span><span class="informations">1594×782 122 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>on my local host my site works perfectly fine. Once upon a time (like two years ago) my github page site also worked fine. Now it doesn’t, so i figured i’d delete it and re-uploaded it under a different repository. Still does not work.</p>
<p>my home page simi works with some styling (no image) but css does not work in any of the other pages, i’m also not seeing any pictures. Finally, some of the links are not correctly directing (for example from about to home) but all of these issues are not present locally? so i’m not sure what is happening.</p>
<p>page: <a href="https://moenegm.github.io/my-simple-site/" class="inline-onebox" rel="noopener nofollow ugc">Mohamed A. Negm</a><br>
repository: <a href="https://github.com/moenegm/my-simple-site" class="inline-onebox" rel="noopener nofollow ugc">GitHub - moenegm/my-simple-site</a></p>

**Comment by JJ** [comment]
<p>Has any rendering error been reported?</p>

**Comment by moenegm** [comment, marked as answer]
<p>there were no errors. I figured it out. I got some help from github support. Apparently i needed to add {…} before my links. so instead of {/CSS/main.css) i need to have {…/CSS/main.css}</p>
